docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -161,10 +162,10 @@ To get playback capability from the client and to set it on the server side, fol
 2. To get playback capability on the client side, use the `GetPlaybackCapability` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
```

Comment:
line 156, 162, 172 and 182. 
How about `SetPlaybackCapability()`, `GetPlaybackCapability()`, `SetMetadata()`, and `GetMetadata()` as other methods, instead of `SetPlaybackCapability`, `GetPlaybackCapability`, `SetMetadata`, and `GetMetadata`?